app/controllers/form_submissions_controller.rb

```diff
@@ -13,4 +13,8 @@ def index
     clear_session_data if current_store.api_response[:status] == 202
   end
 
+  def exclude_save
```

Comment:
This is a weird name - why would you want to exclude save ?  I assume it doesnt really mean that ?